scrapers/trafficpimps.yml

```diff
@@ -0,0 +1,43 @@
+name: trafficpimps
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - pimp.xxx/trailers/
+      - family.xxx/trailers/
+      - petite.xxx/trailers/
+      - cherrypimps.com/
+      - wildoncam.com/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $infoblock: //div[@class="info-block"]
+      $infodata: //div[@class="info-block_data"]
+    scene:
+      Title:
+        selector: //h1[@class="trailer-block_title"]
+        replace:
+          - regex: "LIVE$"
+            with:
+      Date:
+        selector: //div[@class="info-block_data"]/p[@class="text"][1]/text()
+        postProcess:
+          - replace:
+            - regex: (\w+\s)(\w+)(\s)(\d{1,2})(.+)(\d{4})(.+)
+              with: $2 $4, $6
+          - parseDate: January 2, 2006
+      Details: $infoblock/p[@class="text"][1]/text()
+      Tags:
```

Comment:
was there any reason to change the tag and performer selectors?
The original work fine with the scenes i tested
```yaml
      Tags:
        Name: $infoblock/p[contains(text(),'Categories:')]/a
      Performers:
        Name: $infodata/p[contains(text(),'Featuring:')]/a
```

Reply:
It's probably because i made the scraper for pimp or other, then found later it was the same as Cherrypimps scraper, so i probably used my Xpath.